vowpalwabbit/options_boost_po.cc

```diff
@@ -130,6 +135,69 @@ void options_boost_po::add_and_parse(const option_group_definition& group)
     THROW(ex.what());
   }
 }
+  std::vector<std::string> options_boost_po::get_data_values()
```

Comment:
What's the thinking with the rename?

Reply:
remnant of me wanting to use https://www.boost.org/doc/libs/1_37_0/doc/html/boost/program_options/positional_options_desc_id2344626.html but I couldn't get it to work.

Reply:
since I noticed that only data was the valid positional one, there's a way to describe that on boost options api instead of trying to get a single one on parse_args and then checking again if there's any positional and return that as "data" argument. Also it only makes sense to have one single interpretation of all positional arguments, no?